QRCoderTests/PayloadGeneratorTests.cs

```diff
@@ -2887,11 +2893,11 @@ public void contactdata_generator_can_generate_payload_full_vcard4_reverse()
             var note = "Badass programmer.";
             var outputType = PayloadGenerator.ContactData.ContactOutputType.VCard4;
 
-            var generator = new PayloadGenerator.ContactData(outputType, firstname, lastname, nickname, phone, mobilePhone, workPhone, email, birthday, website, street, houseNumber, city, zipCode, country, note, addressOrder: PayloadGenerator.ContactData.AddressOrder.Reversed);
+            var generator = new PayloadGenerator.ContactData(outputType, firstname, lastname, nickname, org, phone, mobilePhone, workPhone, email, birthday, website, street, houseNumber, city, zipCode, country, note, addressOrder: PayloadGenerator.ContactData.AddressOrder.Reversed);
```

Comment:
"org" parameter should be moved to the end. (See comment above for argumentation.)